baseapp/abci.go

```diff
@@ -364,6 +364,10 @@ func (app *BaseApp) halt() {
 
 // snapshot takes a snapshot of the current state and prunes any old snapshottypes.
 func (app *BaseApp) snapshot(height int64) {
+	if app.snapshotManager == nil {
+		app.logger.Error("snapshot manager not configured")
```

Comment:
I think this should be a warning, not an error.

Reply:
I agree on this...

Reply:
Done!